src/models/continuity/ContETFStatic.ts

```diff
@@ -50,7 +54,36 @@ export default class ContETFStatic
     startDot: StuntSheetDot,
     endDot?: StuntSheetDot
   ): void {
-    // TODO: Implement
+    let dx = 0;
+    let dy = 0;
+    switch (this.marchingDirection) {
```

Comment:
If you have some time, can you also handle diagonal directions too? If not can you create an issue so we can remember to do it in the future?

Reply:
Yes, totally forgot that diagonals exist.